frontend/src/components/ProfileContent.tsx

```diff
@@ -1,28 +1,44 @@
 import { useMsal } from '@azure/msal-react';
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { requestMSAuthResult } from '../functions/requestMSAuthResult';
+import { Alert, AlertTitle } from '@mui/material';
 
 export const ProfileContent = () => {
   const { accounts, instance } = useMsal();
   const name = accounts[0] && accounts[0].name;
+  const isValidEmail =
+    /^[A-Za-z0-9._%+-]+@(unsw.edu.au|unswalumni.com|ad.unsw.edu.au|zmail.unsw.edu.au|student.unsw.edu.au)$/.test(
+      accounts[0].username
+    );
+  const [errorMessage, setErrorMessage] = useState<string>('');
 
   useEffect(() => {
     async function fetchData() {
       // TODO: Make sure that the user's email is a UNSW or UNSW Alumni email
       // The user's email is accounts[0].username
       // If not a valid email, don't log the user in
```

Comment:
We can still have a comment here if you like but we can remove the 'TODO' as you've already done it